themes/bootstrap3/js/lib/autocomplete.js

```diff
@@ -269,26 +269,26 @@
             return;
           }
           switch (event.which) {
-            case 9: // tab
```

Comment:
It looks like eslint --fix changed the indentation of this file.

Reply:
Maybe you should open a separate PR to just run `eslint --fix` on existing code so that we can commit these changes separately to avoid confusion.

Reply:
This file is in the lib which isn't usually covered by eslint. Since I added a new lib file, I ran this folder but didn't notice these changes until after commit. I can revert them.

Reply:
I think it makes sense to do this but that should be another PR to avoid clogging this one again.